### ISSUE TITLE
maybe collapse tag clusters that are split across multiple lines?

### ISSUE THREAD

**Issue by felix9** [cheeaun/phanpy#214]
some people post tags on separate lines, like this:
https://bark.lgbt/@henrychinaski/110887703353767939

**Comment by felix9** [comment]
another example, multiple tags on a series of lines:
https://bbq.snoot.com/@NoelJPenaflor/110890644596490775

**Comment by cheeaun** [comment]
@felix9 yeah the logic now doesn't apply to the first paragraph, to be safe, as it might be an actual post content instead of hashtag stuffing/tag clusters.
